core/src/test/java/de/unistuttgart/iaas/amyassist/amy/core/pluginloader/PluginManagerTest.java

```diff
@@ -49,16 +52,21 @@
 class PluginManagerTest {
 	@Reference
 	private TestFramework testFramework;
+
 	private PluginManager serviceUnderTest;
 	private Properties properties;
 
 	@BeforeEach
 	void setup() {
 		this.testFramework.mockService(DependencyInjection.class);
 		this.testFramework.mockService(PluginLoader.class);
+		this.testFramework.mockService(CommandLineArgumentHandler.class);
 		ConfigurationLoader configurationLoader = this.testFramework.mockService(ConfigurationLoader.class);
+		Environment environment = this.testFramework.mockService(Environment.class);
+		when(environment.getWorkingDirectory()).thenReturn(Paths.get("").toAbsolutePath());
+
 		this.properties = new Properties();
-		when(configurationLoader.load("plugin.config")).thenReturn(this.properties);
+		configurationLoader.store("plugin.config", this.properties);
```

Comment:
that seems wrong to me

Reply:
i will review this a second time

Reply:
Git blame says this was your change @Legion2. But I agree, it seems weird, and I think the first solution was better